<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accounts</class>
 <widget class="QWidget" name="Accounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="AccountsView" name="accountsView">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="indentation">
        <number>0</number>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="publicTimelineLabel">
         <property name="text">
          <string>public timeline:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="publicTimelineComboBox">
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Twitter</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Identi.ca</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Both</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QToolButton" name="addAccountButton">
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icons/add_48.png</normaloff>:/icons/add_48.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteAccountButton">
       <property name="icon">
        <iconset resource="../res/resources.qrc">
         <normaloff>:/icons/cancel_48.png</normaloff>:/icons/cancel_48.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountsView</class>
   <extends>QTreeView</extends>
   <header>accountsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
